auto_nag/utils.py

```diff
@@ -471,3 +471,48 @@ def get_human_lag(date):
     dt = dateutil.parser.parse(date)
 
     return humanize.naturaldelta(today - dt)
+
+
+def get_version(jsonVersion, key):
+    # In X.Y, we just need X
+    version = jsonVersion[key].split('.')
+    return version[0]
+
+
+def get_current_versions():
```

Comment:
Nit: you could use libmozdata here

Reply:
Yes you're right: I almost just moved the code.
So let's do that in an other commit